src/promptflow/promptflow/_sdk/_orm/trace.py

```diff
@@ -100,3 +106,45 @@ def list(
             if len(current_spans) > 0:
                 line_runs.append(copy.deepcopy(current_spans))
             return line_runs
+
+    @staticmethod
+    def _construct_query_with_line_run_id(session: Session, line_run_id: str) -> Query:
+        # TODO: maybe we need a LineRun table
```

Comment:
MARK: when we successfully generate a line run from a group of spans, it should not change under current design - maybe we can persist this for further query.